src/styles/landing-page.less

```diff
@@ -44,14 +44,19 @@
   background-color: @landing-side-bar-bg;
   color: @color-pf-white;
   @media(min-width: @screen-sm-min) {
-    flex: 0 1 auto;
-    width: 325px;
+    bottom: 0;
+    overflow: hidden;
+    overflow-y: auto;
+    position: fixed;
+    right: 0;
```

Comment:
> Setting the right position to 15px shows the scrollbar.

When there is no scrollbar, is there a right gutter? Are different widths needed for different browsers and platforms since they style scrollbars differently?